packages/artillery/test/cli/run-typescript.test.js

```diff
@@ -99,14 +99,13 @@ tap.test('Runs correctly when package is marked as external', async (t) => {
 tap.test(
   'Failure from a Typescript processor has a resolvable stack trace via source maps',
   async (t) => {
-    const [exitCode, output] = await execute([
+    const [_exitCode, output] = await execute([
       'run',
       '-o',
       `${reportFilePath}`,
       'test/scripts/scenarios-typescript/error.yml'
     ]);
 
-    t.equal(exitCode, 11, 'CLI should exit with code 11');
```

Comment:
Why was the check for a failure exitCode removed? Maybe it shouldn't explicitly be 11, but it should still be non-zero, no?

Reply:
Exit code 11 was not explicitly set anywhere by Artillery itself either btw, it seemed to be set by Node.js when a worker crashes with an unhandled exception.

Reply:
Ah I see. Possibly this test should now check the number of vusers.failed then

Reply:
yep, that'd be a good test but does not belong in this specific test suite, as it's a TypeScript-specific test cases, not tests for the hooks API/behavior

Reply:
yeah I was wondering the same tbh